src/test/feature/ManagementTool/test_hawq_register.cpp

```diff
@@ -369,3 +369,25 @@ TEST_F(TestHawqRegister, TestUsage2Behavior2) {
   util.execute("drop table simple_register_table;");
   util.execute("drop table new_simple_register_table;");
 }
+
+TEST_F(TestHawqRegister, TestUsage2ForceOption) {
+  SQLUtility util;
+  util.execute("drop table if exists simple_register_table_force;");
+  util.execute("drop table if exists new_simple_register_table_force;");
+  util.execute("create table simple_register_table_force(i int) with (appendonly=true, orientation=row) distributed randomly;");
+  util.execute("insert into simple_register_table_force values(1), (2), (3);");
+  EXPECT_EQ(0, Command::getCommandStatus("hawq extract -d " + (string) HAWQ_DB + " -o tmp.yml testhawqregister_testusage2forceoption.simple_register_table_force"));
+  EXPECT_EQ(0, Command::getCommandStatus("hawq register -d " + (string) HAWQ_DB + " -c tmp.yml testhawqregister_testusage2forceoption.new_simple_register_table_force"));
+  util.query("select * from new_simple_register_table_force;", 3);
+
+  util.execute("drop table if exists simple_register_table_force;");
+  util.execute("create table simple_register_table_force(i int) with (appendonly=true, orientation=row) distributed randomly;");
+  util.execute("insert into simple_register_table_force values(4), (5), (6), (7);");
+  EXPECT_EQ(0, Command::getCommandStatus("hawq extract -d " + (string) HAWQ_DB + " -o tmp.yml testhawqregister_testusage2forceoption.simple_register_table_force"));
+  EXPECT_EQ(0, Command::getCommandStatus("hawq register -d " + (string) HAWQ_DB + " -c tmp.yml --force testhawqregister_testusage2forceoption.new_simple_register_table_force"));
+  util.query("select * from new_simple_register_table_force;", 4);
```

Comment:
Should here throw error, indicating the existing files for table 'new_simple_register_table_force' are not included in .yml file???